ov/logical_interconnects.go

```diff
@@ -0,0 +1,834 @@
+	Description                          utils.Nstring           `json:"description,omitempty"`                 //"description": null,
+	DomainUri                            utils.Nstring           `json:"domainUri"`                             //"domainUri": "/rest/domains/7b655ba0-11c1-4e9c-a191-672e1d0bbd8e",
+	ETAG                                 string                  `json:"eTag,omitempty"`                        //"eTag": "bedd16b1-1d27-4ecd-b483-407635de7431",
+	EnclosureType                        string                  `json:"enclosureType,omitempty"`               //"enclosureType": "SY12000",
+	InternalNetworkUris                  []utils.Nstring         `json:"internalNetworkUris,omitempty"`         //"internalNetworkUris": ["/rest/ethernet-networks/cbde97d0-c8f1-4aba-aa86-2b4e5d080401"],
+	FabricUri                            utils.Nstring           `json:"fabricUri,omitempty"`                   //"fabricUri": "/rest/fabrics/c2fc09bc-323f-40b4-8a7e-d99ca28df969",
+	QosConfiguration                     *QosConfiguration       `json:"qosConfiguration,omitempty"`            // "qosConfiguration": {},
+	IcmLicenses                          *IcmLicenses            `json:"icmLicenses,omitempty"`                 //"icmLicenses":{...},
+	EnclosureUris                        []utils.Nstring         `json:"enclosureUris,omitempty"`               //"enclosureUris": ["/rest/enclosures/013645CN759000AC"],
+	FusionDomainUri                      utils.Nstring           `json:"fusionDomainUri,omitempty"`             //"fusionDomainUri": "/rest/domains/7b655ba0-11c1-4e9c-a191-672e1d0bbd8e",
+	EthernetSettings                     *EthernetSettings       `json:"ethernetSettings,omitempty"`            //"ethernetSettings": {...},
+	LogicalInterconnectGroupUri          utils.Nstring           `json:"logicalInterconnectGroupUri,omitempty"` //"logicalInterconnectGroupUri": "/rest/logical-interconnect-groups/3de143b3-bf77-4944-aa31-18084676b664",
+	StackingHealth                       string                  `json:"stackingHealthomitempty"`               //"stackingHealth": "BiConnected",
+	InterconnectMap                      *InterconnectMap        `json:"interconnectMap,omitempty"`             //"interconnectMap": {...},
+	PortMonitor                          *PortMonitor            `json:"portMonitor,omitempty"`                 //"portMonitor": {...},
+	SnmpConfiguration                    *SnmpConfiguration      `json:"snmpConfiguration,omitempty"`           // "snmpConfiguration": {...},
+	TelemetryConfiguration               *TelemetryConfiguration `json:"telemetryConfiguration,omitempty"`      // "telemetryConfiguration": {...},
+	ScopeUri                             string                  `json:"ScopeUri,omitempty"`                    //"ScopeUri": null,
+	SecurityStandardMode                 string                  `json:"ScopeUri,omitempty"`                    //"securityStandardMode": null,
```

Comment:
the json for securityStandardMode should be securityStandardMode and in the api documentation, its scopesUri attribute. Please verify all the other fields